plugins/modules/iso_extract.py

```diff
@@ -157,7 +164,8 @@ def main():
         cmd = [binary, 'x', image, '-o%s' % tmp_dir] + extract_files
     else:
         cmd = [module.get_bin_path('mount'), '-o', 'loop,ro', image, tmp_dir]
-
+    if password:
```

Comment:
I don't think `mount` will accept this, so this `if` needs to be moved into the `if binary:` above.